addons/pos_restaurant/static/src/app/floor_screen/table.js

```diff
@@ -29,13 +29,14 @@ export class Table extends Component {
             containerHeight: 0,
             containerWidth: 0,
         });
+        this.table = this.pos.models["restaurant.table"].get(this.props.tableId);
```

Comment:
Instead of `tableId` as props, give `table` record as props. That way we can keep keep using the existing `this.props.table`.

Reply:
Yes mybad, it was a test :)